ledfx/api/virtual_effects.py

```
@@ -295,14 +296,41 @@ async def delete(self, virtual_id) -> web.Response:
+
+        _LOGGER.warning(
+            f"Deleting effect {effect_type} for virtual {virtual_id}"
+        )
+
+        if effect_type is None:
+            # top level active effect clear
+            virtual.clear_effect()
+
+            for virtual_cfg in self._ledfx.config["virtuals"]:
+                if virtual_cfg["id"] == virtual_id:
+                    if "effect" in virtual_cfg:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)